packages/form-field/src/FormField/FormField.tsx

```diff
@@ -66,6 +66,7 @@ export const FormField = forwardRef<HTMLDivElement, FormFieldProps>(
               {label && (
                 <Label
                   data-testid="lg-form_field-label"
+                  data-lgid="lg-form_field-label"
```

Comment:
should these somehow be responsive to the lgId prop?

Reply:
So it's lookign for the label that describes the element you already have? if so, that works